exasol_transformers_extension/udfs/models/token_classification_udf.py

```diff
@@ -159,7 +158,7 @@ def create_dataframes_from_predictions(
                                    f"Prediction results contain columns: {result_df_column_names}") from e
             else:
                 # if the result for an input is empty, just append an empty result df, and the input will be dropped during concatenation
-                result_df = pd.DataFrame(result)
+                result_df = pd.DataFrame({})
```

Comment:
the dataframe still needs the correct columns

Reply:
if the result is empty it does not have them either way. these results are later dropped before anything with the columns is done. this is basically just a placeholder to make sure we drop the right input together with the empty output.